server/models/review-request.js

```diff
@@ -80,6 +80,12 @@ const requestSchema = new mongoose.Schema({
     enum: ["pending", "declined", "accepted", "closed"],
     default: "pending",
   },
+
+  // comment for selected reviewer
+  comment: {
+    type: String,
+    default: "",
+  },
```

Comment:
Why can't we save comments to the user model or u plan to show them on the reviews page? This way we can remove fetchComments in profile.js and just fetch user by Id in handleInitState.